packages/nextjs/next.config.js

```diff
@@ -9,6 +9,18 @@ const nextConfig = {
   eslint: {
     ignoreDuringBuilds: process.env.NEXT_PUBLIC_IGNORE_BUILD_ERROR === "true",
   },
+  serverRuntimeConfig: {
+    providerConfig: {
+      networks: [
+          { name: "mainnet", rpcUrl: "https://eth.drpc.org" },
+          { name: "sepolia", rpcUrl: "https://sepolia.drpc.org", registry: "0x03d5003bf0e79c5f5223588f347eba39afbc3818" }
+      ]
+    },
+    mongoConfig: {
+      connectionStr: "mongodb+srv://verinta:BFWmxukoOsNdIx4x@ceptorclub.rq4oohp.mongodb.net/",
+      DB: "CeptorClub_allan"
```

Comment:
use the test database in current branch, switch to the production database when merge

Reply:
Hey @allanma88 I think it would be better if you make this an ENV var. So we could properly configure the CI/CD for each environment (production/preview).

Besides, it's a good practice to not have `keys` exposed in the repo.